datafusion/core/src/physical_planner.rs

```diff
@@ -2242,7 +2242,7 @@ mod tests {
                 dict_id: 0, \
                 dict_is_ordered: false, \
                 metadata: {} } }\
-        ], metadata: {}, functional_dependencies: FunctionalDependencies { deps: [] } }, \
+        ], metadata: {}, functional_dependencies: FunctionalDependencies { deps: [] }, fields_map: {\"a\": 0} }, \
```

Comment:
I suppose there's no easy way to omit this new field from debug :slightly_frowning_face: (other than newtyping it?)

Not a big deal I guess

Reply:
The `Debug` is derived, the only option I see is to implement `Debug` trait manually for `DFSchema`.
I'll create a followup on this.